inspirehep/base/templates/search/searchbar_frame_base.html

```diff
@@ -0,0 +1,34 @@
+{#
```

Comment:
I'm overwriting https://github.com/inspirehep/invenio-search/blob/master/invenio_search/templates/search/searchbar_frame_base.html#L29 and removing that line to no longer show the search bar above the search results